src/main/java/com/mentalhealthapi/mentalhealthapi/service/BlogServiceStub.java

```diff
@@ -6,11 +6,11 @@
 import java.util.List;
 
 public class BlogServiceStub implements IBlogService{
-
     /**
      * Fetch all blogs - regardless the user
      *
      * @return a list of blogs
+     *
```

Comment:
Multiple line comments could have an extra line to better match the examples on medium.com. https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3